google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -2638,6 +2639,8 @@ public Builder setRetentionPeriodDuration(Duration retentionPeriod) {
      */
     public abstract Builder setIpFilter(IpFilter ipFilter);
 
+    abstract Builder setIsUnreachable(Boolean isUnreachable);
```

Comment:
this should be public, so customers can set it in any tests they might want to write.